AtData/AtPadCharge.cxx

```diff
@@ -0,0 +1,18 @@
+#include "AtPadCharge.h"
+
+std::unique_ptr<AtPad> AtPadCharge::Clone()
+{
+   return std::make_unique<AtPadCharge>(*this);
+}
+
+const AtPadCharge::traceElec &AtPadCharge::GetElectrons() const
```

Comment:
Should be in header, not cxx file (trivial getters/setter) (https://github.com/ATTPC/ATTPCROOTv2/wiki/Coding-conventions#header-files)